pkg/porter/runs_test.go

```diff
@@ -0,0 +1,54 @@
+package porter
+
+import (
+	"get.porter.sh/porter/pkg/claims"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestPorter_ListInstallationRuns(t *testing.T) {
```

Comment:
This is a good test. Let's add one more testcase for checking the formatted command output.

The tests for `porter installation show` in pkg/porter/show_test.go demonstrate how to write a test that checks that the printed output looks the way that you expect. It captures the output from the command, and then compares it against expected output in a file.

The `p.CompareGoldenFile` function assists with updating the contents of those test files to match the current test output (which is helpful when you intentionally change the output of those commands). Once the test files exist, you can run `mage updateTestFiles` and all of those test files will be synced with the current output.

The tricky part with those tests is making sure that timestamps do not cause the output to be different every time that you run the test. So the test uses fixed timestamps to make comparing the outputs easier.

```
i := p.TestClaims.CreateInstallation(claims.NewInstallation("dev", "mywordpress"), p.TestClaims.SetMutableInstallationValues
```

The SetMutableInstallationValues function (there is also SetMutableRunValues, and SetMutableResultValues) updates the timestamps and IDs so that the tests are reproducible.